Makefile

```diff
@@ -100,5 +107,9 @@ test-coverage:
 	@set -e; \
 	./scripts/coverage.sh
 
+repeat=20000
+steps=50
+heap-pages=2048
+db-cache=1024
 benchmark:
-	@GOMAXPROCS=1 go test --tags=nonwasmenv -run=XXX -bench=. -benchtime=2000x ./runtime/...
\ No newline at end of file
+	@GOMAXPROCS=1 go test --tags="nonwasmenv" -run=XXX -bench=. -benchtime=$(repeat)x ./runtime/... -steps=$(steps) -repeat=$(repeat) -heap-pages=$(heap-pages) -db-cache=$(db-cache);
```

Comment:
do we still need to pass `-benchtime=20x` ?

Reply:
for some reason I couldn't override testing.N global value, so I was getting inconsistent results at places. probably needs some debugging. i will check it again